scripts/exglobal_stage_ic.py

```diff
@@ -37,6 +37,8 @@ def main():
     for key in stage.task_config.keys():
         if key.startswith("COM"):
             stage_dict[key] = stage.task_config[key]
+        if key == "ENSMEM":
+            stage_dict[key] = stage.task_config[key]
```

Comment:
Use `in` so you don't have to add an extra test in the loop:

```suggestion

        if "ENSMEM" in stage.task_config:
            stage_dict["ENSMEM"] = stage.task_config["ENSMEM"]
```

Alternatively, you could use a ternary, though it's not as intuitive:
```python
        stage_dict["ENSMEM"] = stage.task_config["ENSMEM"] if "ENSMEM" in stage.task_config else 0
```